chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -543,14 +546,47 @@
                   target_platform_version: target_platform_version)
       end
 
-      missing_files = !File.exist?("#{os_dir}/install/boot/#{arch}/common")
+      if target_platform_distro == "suse"
+        missing_files = !File.exist?("#{os_dir}/install/boot/#{arch}/common")
+      elsif target_platform_distro == "opensuse"
+        # if we use remote URL, then we don't require a rsync of the install
+        # media, but we still need to copy the kernel/initrd
+        install_url = params["install_url"]
+
+        missing_files = !File.exist?("#{os_dir}/install/boot/#{arch}/common")
+
+        if missing_files && !install_url.nil? && !install_url.empty?
+          kernel_path = Pathname.new(os_dir).join("install").join(kernel)
+          kernel_tmp_path = Pathname.new(os_dir).join("install").join("#{kernel}.check")
+          initrd_path = Pathname.new(os_dir).join("install").join(initrd)
+          initrd_tmp_path = Pathname.new(os_dir).join("install").join("#{initrd}.check")
+
+          unless kernel_path.exist?
```

Comment:
Metrics/BlockNesting: Avoid more than 3 levels of block nesting. (https://github.com/bbatsov/ruby-style-guide#three-is-the-number-thou-shalt-count)